snuba/web/views.py

```diff
@@ -459,10 +459,6 @@ def drop(*, dataset: Dataset):
         redis_client.flushdb()
         return ("ok", 200, {"Content-Type": "text/plain"})
 
-    @application.route("/tests/error")
-    def error():
-        1 / 0
-
```

Comment:
Are you sure it is not used in sentry tests ?

Reply:
Yeah I couldn't find it referenced anywhere. Also ran the Snuba test suite to be sure.